### DISCUSSION TITLE
Default Hourly rate

### DISCUSSION THREAD

**Discussion by ctpntn** [kimai/kimai#2218] [Questions & Answers]
How setup up a default hourly rate?

**Comment by kevinpapst** [comment]
In the User preferences there is an hourly rate, which will be used for every record as last fallback if no "fee" config for customer/project/activity is defined.

See https://www.kimai.org/documentation/timesheet.html - chapter "Rate Calculation"

I will create a dedicated documentation page for rates soon.

**Comment by ctpntn** [comment]
Yes, I see. But if I've a lot of user and I want every user have a default value. Do I have to modify every single user?

**Reply by kevinpapst** [reply]
Yes, for now there is no global "default hourly rate". But sounds like a good feature request!

**Comment by limitedAtonement** [comment]
I have been using Kimai for several years. Really excellent project. I always have entered the hourly rate on every new timesheet. I'm now seeing that there is a way to set default hourly rates on users, projects, etc. In my `/en/admin/project/12/details`, I don't see any "Hourly rate" as described here: https://www.kimai.org/documentation/project.html#rates-example. I have a "Prices" section. Is that the same thing? Should I create a ticket to update the documentation?

**Reply by kevinpapst** [reply]
Oh, the screenshot title is outdated. 
Yes, it is "Prices" now. I dont know why, but nobody understood "Hourly rate". So I change dit.

I updated the documentation and hope, it reflects reality better now 😁 
Please let me know if you stumble upon such mistakes.

**Reply by kevinpapst** [reply]
And wow: you really set the price manually, every time? Time to upgrade your workflow 👍 🚀 

Use your user preference as base price. Overwrite it per customer if you have special rates for some of them.

**Reply by limitedAtonement** [reply]
Great, thanks for the documentation update.

No kidding, the workflow was sooo bad. I input new Prices everywhere. It will be sooo nice!
